scanners/macos/macos.go

```diff
@@ -60,7 +60,7 @@ func (Scanner) DefaultOptions() models.OptionNode {
 }
 
 // Configs ...
-func (scanner *Scanner) Configs() (models.BitriseConfigMap, error) {
+func (scanner *Scanner) Configs(_ bool) (models.BitriseConfigMap, error) {
```

Comment:
TIL that you can use the blank identifier in the function signature as well. Nice.